src/applications/mhv-landing-page/sass/mhv-landing-page.scss

```diff
@@ -115,3 +115,40 @@ $anchor-font-size: 1.1875rem;
     justify-content: space-between;
   }
 }
+
+// Match the va-alert styling.
+.mhv-c-reg-alert {
+  border-left: 0.5rem solid;
```

Comment:
Note: Most of these values were taken as is from the VADS alert. Different unit types are used and even the font sizing is different from standard VADS sizing.